migrations/1665416411399_allow-profile-picture-field-when-creating-a-user.js

```diff
@@ -0,0 +1,7 @@
+exports.shorthands = undefined
```

Comment:
let's try to ensure the name of the migration is about the change happening in the database

something like 

`add_profilePictureUrl_column_to_users_table` or something better